.github/workflows/ci.yaml

```diff
@@ -96,8 +96,35 @@ jobs:
           ./project/scripts/sbt ";dotty-bootstrapped/compile ;dotty-bootstrapped/test;sjsSandbox/run;sjsSandbox/test;sjsJUnitTests/test;sjsCompilerTests/test ;configureIDE"
           ./project/scripts/bootstrapCmdTests
 
+
+  test-windows:
+    runs-on: [self-hosted, Windows]
+
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v2
+
+      - name: Test
+        run: sbt ";compile ;test"
+        shell: cmd
+
+  test_bootstrapped-windows:
+    runs-on: [self-hosted, Windows]
+
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v2
+
+      - name: Test
+        run: sbt ";dotty-bootstrapped/compile ;dotty-bootstrapped/test"
+        shell: cmd
+
+      # - name: Scala.js Test
+      #   run: sbt ";sjsJUnitTests/test"
+      #   shell: cmd
```

Comment:
Why is that disabled?

Reply:
It causes a failure: https://github.com/lampepfl/dotty/runs/1225047885?check_suite_focus=true

Do you see clue from the message?

Reply:
Yeah ^^
```
Caused by: java.io.IOException: Cannot run program "node": CreateProcess error=2, The system cannot find the file specified
```
We need to install Node.js ;)

Reply:
Cool, thanks. I'm making a PR to enable it.